### PR TITLE
Fixing deprecated "Passing null to parameter"

### PR DESCRIPTION
**Pull request by RafaMelazzo**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| **Issue**           | https://mundipagg.atlassian.net/browse/PAOPN-355
| **What?**         | Fixed deprecated "Passing null to parameter" on functions in PHP8
| **Why?**          | To avoid exceptions ion this situations
| **How?**          | By adding `?? ''` to functions that expect strings

### CODE REVIEW

File: Concrete/Magento2CoreSetup.php [hunk 1]
```diff
@@ -438,17 +438,17 @@ static private function getBrandConfig($storeConfig, $section)
         $selectedBrands = $storeConfig->getValue($section .  'cctypes', $scope, $storeId) ?? "";
         $brands = array_merge([''], explode(
             ',',
-            $selectedBrands
+            $selectedBrands ?? ''
```
Comment (fabiano-mallmann) [inline Concrete/Magento2CoreSetup.php:441]: This condition is already handled in line 438

Comment (RafaMelazzo) [reply Concrete/Magento2CoreSetup.php]: Reversed! Thanks!

File: Helper/ModuleHelper.php
Comment (fabiano-mallmann) [inline Helper/ModuleHelper.php:1]: Acredito que esse não seja tão necessário, visto que na linha acima ele verifica se não for nulo.

**Review by mauriciohaygert — APPROVED** [review]

File: Helper/ModuleHelper.php
Comment (RafaMelazzo) [reply Helper/ModuleHelper.php:1]: Arrumado

**Review by fabiano-mallmann — APPROVED** [review]

**Pull request merged**